adapters/adoppler/adopplertest/exemplary/no-bid.json

```diff
@@ -1,13 +1,50 @@
-{"mockBidRequest": {"id": "req1",
-                    "imp":[{"id": "imp1",
-                            "banner": {"w": 100,
-                                       "h": 200},
-                            "ext": {"bidder": {"adunit": "unit1"}}}]},
- "httpCalls": [{"expectedRequest": {"uri": "http://adoppler.com/processHeaderBid/unit1",
-                                    "body": {"id": "req1-unit1",
-                                        "imp": [{"id": "imp1",
-                                                 "banner": {"w": 100, "h": 200},
-                                                 "ext": {"bidder": {"adunit": "unit1"}}}]}},
-                "mockResponse": {"status": 204,
-                                 "body": ""}}],
- "expectedBidResponses": []}
+{
+  "mockBidRequest":{
+    "id":"req1",
+    "imp":[
+      {
+        "id":"imp1",
+        "banner":{
+          "w":100,
+          "h":200
+        },
+        "ext":{
+          "bidder":{
+            "adunit":"unit1"
+          }
+        }
+      }
+    ]
+  },
+  "httpCalls":[
+    {
+      "expectedRequest":{
+        "uri":"http://app.trustedmarketplace.com/processHeaderBid/unit1",
+        "body":{
+          "id":"req1-unit1",
+          "imp":[
+            {
+              "id":"imp1",
+              "banner":{
+                "w":100,
+                "h":200
+              },
+              "ext":{
+                "bidder":{
+                  "adunit":"unit1"
+                }
+              }
```

Comment:
Test cases that are testing non 200 response are expected to be added to the `supplemental` directory. Please consider moving this one there